src/app/games/kamisado/kamisado.component.html

```diff
@@ -5,7 +5,7 @@
      preserveAspectRatio="xMidYMid meet"
 >
     <g *ngFor="let line of board; index as y">
-        <g *ngFor="let caseContent of line; index as x">
+        <g *ngFor="let spaceContent of line; index as x">
```

Comment:
j'ai l'impression que tu fais le renommage case -> space dans chaque ticket :D Y'en a encore beaucoup après ?
D'ailleurs y'a des space qui devraient être des square, notamment pour kamisado

Reply:
Là j'ai enlevé l'entierté absolue, mais ouais j'en avais p-e enlevé par-ci par-là puis arrêté (yen avais genre, 1200)
Ouais j'ai enlevé l'inacceptable pour mettre du générique, tu veux qu'on tickette "space -> square for square-game ?" ?

Reply:
Yes plz!